File: JAVA/Problema1015.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
		Scanner scn = new Scanner(System.in);
		
		double x1 = scn.nextDouble();
		double y1 = scn.nextDouble();
		double x2 = scn.nextDouble();
		double y2 = scn.nextDouble();
		
		double p1 = (x2-x1)*(x2-x1);
		double p2 = (y2-y1)*(y2-y1);
		
		double dist = Math.sqrt(p1 + p2);
		
		System.out.printf("%.4f\n", + dist);
    }
 
}

File: C/Problema1015.c
#include <stdio.h>
#include <math.h>

int main(){
	float  x, y, x1, y1, x2, y2;

	scanf("%f %f", &x1, &y1);
	scanf("%f %f", &x2, &y2);
	x = pow(x2-x1, 2);
	y = pow(y2-y1, 2);
	double p = x+y;
	float d = sqrt(p);

	printf("%.4f\n", d);
}
